master/internal/hpimportance/hpimportance.go

```diff
@@ -1,15 +1,241 @@
 package hpimportance
 
+/**
```

Comment:
these comments usually go above the package declaration in go https://github.com/golang/go/blob/master/src/crypto/cipher/cbc.go#L1-L12